src/nls/fi/strings.js

```diff
@@ -767,8 +774,11 @@ define({
     "DESCRIPTION_FONT_SMOOTHING"                     : "Vain Mac: ota käyttöön alipikselien reunapehmennys valinnalla ”subpixel-antialiased” tai harmaansävyinen reunapehmennys valinnalla ”antialiased”",
     "DESCRIPTION_OPEN_PREFS_IN_SPLIT_VIEW"           : "false: poista asetustiedoston avaaminen käytöstä jaetussa näkymässä",
     "DESCRIPTION_OPEN_USER_PREFS_IN_SECOND_PANE"     : "false: avaa käyttäjän asetustiedosto vasemmassa tai ylemmässä ruudussa",
+    "DESCRIPTION_MERGE_PANES_WHEN_LAST_FILE_CLOSED"  : "true: yhdistä paneelit kun viimeinen tiedosto paneelista on suljettu käyttäen paneelin yläosassa olevaa suljin painiketta",
+    "DESCRIPTION_SHOW_PANE_HEADER_BUTTONS"           : "Valitse koska paneelin yläosassa näytetään suljin- ja flip-view-painikkeita",
     "DEFAULT_PREFERENCES_JSON_HEADER_COMMENT"        : "/*\n * Tämä on vain luku -tiedosto {APP_NAME}in tukemista\n * asetuksista.\n * Käytä tätä apuna, kun muutat asetustiedostoasi\n * ”brackets.json”, joka on avattu toiseen ruutuun.\n * Lisätietoa siitä, kuinka käyttää asetuksia {APP_NAME}issa,\n * löydät verkkosivulta osoitteessa https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#preferences\n */",
-    "DEFAULT_PREFERENCES_JSON_DEFAULT"               : "Oletus"
+    "DEFAULT_PREFERENCES_JSON_DEFAULT"               : "Oletus",
+    "DESCRIPTION_PURE_CODING_SURFACE"                : "true: näytä vain koodinäkymä ja piilota muut UI elementit {APP_NAME}issa"
```

Comment:
“true: ota vain koodi \u2011näkymä käyttöön ja piilota muut {APP_NAME}in käyttöliittymäelementit”